test/testcafe/framework/page-objects/DeviceCodeActivatePageObject.js

```diff
@@ -52,15 +59,31 @@ export default class DeviceCodeActivatePageObject extends BasePageObject {
     return this.form.setTextBoxValue('credentials.passcode', value);
   }
 
+  getTerminalTitle() {
+    if (userVariables.v3) {
+      return this.form.getAlertBoxText();
+    }
+    return this.getFormTitle();
+  }
+
   getTerminalContent(){
+    if (userVariables.v3) {
+      return this.getPageSubtitle();
+    }
     return this.form.getTerminalContent();
   }
 
   isTerminalSuccessIconPresent() {
+    if (userVariables.v3) {
+      return this.form.getElement('.infobox-success').exists && this.form.getElement('.MuiAlert-icon').exists;
+    }
```

Comment:
To determine if it is the success info box for v3, I would replicate what is being done in this file `test/testcafe/framework/page-objects/components/CalloutObject.js` in this function `isError` but instead of `infobox-error` as the class, it would be `infobox-success`. We want to stay away from asserting on MUI classnames in the case we ever move away from MUI.

Reply:
Makes sense in terms of not wanting to assert on MUI classnames, I only did that because it doesn't seem like there are any non-MUI selectors that I can target for the icon itself and this function is supposed to look specifically for a success icon. 

<img width="1834" alt="Screenshot 2023-09-12 at 4 21 47 PM" src="https://github.com/okta/okta-signin-widget/assets/106110543/43965420-81d3-4e8a-a31e-cea627f16801">

That being said, I feel like looking for an `infobox-success` and `infobox-error` classname is sufficient for the v3 version of this function because those components should always come with their respective icons, otherwise the error is bigger than just this spec, do you agree? If so, something along the lines of either `this.form.elementExist('.infobox-success')` or `this.form.getAlertBox().hasClass('infobox-success')` should be sufficient.

Reply:
^ Yep! Totally agree with the last statement 🙏